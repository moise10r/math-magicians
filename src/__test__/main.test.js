import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Main from '../components/main';

const MockMain = () => (
  <BrowserRouter>
    <Main />
  </BrowserRouter>
);
it('should match the main snapshot', () => {
  const tree = renderer.create(<MockMain />).toJSON();
  expect(tree).toMatchSnapshot();
});