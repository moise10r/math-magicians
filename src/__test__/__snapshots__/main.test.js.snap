// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the main snapshot 1`] = `
<div
  className="main-home-container"
>
  <header
    className="header-main-container"
  >
    <div
      className="main-header-wrapper flex-between"
    >
      <div
        className="logo-wrapper"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Math
          <span>
            Magicians
          </span>
        </a>
      </div>
      <ul
        className="link-list"
      >
        <li
          className="link"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="link"
        >
          <a
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li
          className="link"
        >
          <a
            href="/quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </div>
  </header>
  <div
    className="main-home-container"
  >
    <section
      className="main-headline-section"
    >
      <div
        className="bg"
      />
      <div
        className="main-headline-container flex-center"
      >
        <div
          className="main-wrapper"
        >
          <h2>
            The Best
            <span>
              Calculator
            </span>
            Out There
          </h2>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec malesuada lorem maximus mauris scelerisque, at rutrum nulla dictum. Ut ac ligula sapien. Suspendisse cursus faucibus finibus.
          </p>
          <a
            href="#2"
          >
            Read More
          </a>
        </div>
      </div>
    </section>
  </div>
</div>
`;
